data/manual-published-branches.yaml

```diff
@@ -15,8 +15,8 @@ version:
     - '3.6'
     - '3.4'
     - '3.2'
-  stable: '4.0'
-  upcoming: '4.2'
+  stable: '4.2'
+  upcoming: null
 git:
   branches:
     manual: 'v4.0'
```

Comment:
Do we need to change this as well?

Reply:
Heh -- I updated.  But actually, I don't actually think it does anything, but I guess today/tomorrow will not be the day we find out by not changing it 😄